gapis/trace/android/adreno/profiling_data.go

```diff
@@ -196,6 +196,10 @@ func processGpuSlices(ctx context.Context, processor *perfetto.Processor, captur
 			Value: &service.ProfilingData_GpuSlices_Slice_Extra_IntValue{IntValue: uint64(hwQueueIds[i])},
 		})
 
+		if names[i] == "Surface" && groupIds[i] != -1 {
```

Comment:
We probably can't rely on the name

Reply:
Long term, no. This code is specific to the current Adreno producers though.